VisualPinball.Engine/VPT/Trough/TroughData.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Visual Pinball Engine
+// Copyright (C) 2020 freezy and VPE Team
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+#region ReSharper
+// ReSharper disable UnassignedField.Global
+// ReSharper disable StringLiteralTypo
+// ReSharper disable FieldCanBeMadeReadOnly.Global
+// ReSharper disable ConvertToConstant.Global
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using VisualPinball.Engine.IO;
+using VisualPinball.Engine.VPT.Table;
+
+namespace VisualPinball.Engine.VPT.Trough
+{
+	[Serializable]
+	public class TroughData : ItemData
+	{
+		public override string GetName() => Name;
+		public override void SetName(string name) { Name = name; }
+
+		[BiffString("NAME", IsWideString = true, Pos = 9)]
+		public string Name;
+
+		[BiffString("ENTK", Pos = 1)]
+		public string EntryKicker = string.Empty;
+
+		[BiffString("EXIT", Pos = 10)]

```

Comment:
Not really sure how `Pos` works, but is `6`, `7`, and `8` skipped for a reason?

Reply:
No reason.

The goal of `Pos` is to write the attributes in the same order than VP. It made binary comparisons easier when I was implementing it. They are just a sort key, so skipping isn't a problem, but I'll update them nevertheless.